src/universal/modules/meeting/components/MeetingUpdates/MeetingUpdates.js

```diff
@@ -1,6 +1,6 @@
 import React, {PropTypes} from 'react';
 import withStyles from 'universal/styles/withStyles';
-import {css} from 'aphrodite/no-important';
+import {css} from 'aphrodite-local-styles/no-important';
```

Comment:
I am seeing the following on this file:

```
client.js:58 [HMR] connected
client.js:105 [HMR] bundle has warnings:
client.js:107 [HMR] AUTH0_CLIENT_ID environment variable is undefined.
client.js:107 [HMR] AUTH0_DOMAIN environment variable is undefined.
client.js:107 [HMR] HOST environment variable is undefined.
client.js:107 [HMR] PORT environment variable is undefined.
warning.js:36 Warning: Failed prop type: Required prop `meetingPhase` was not specified in `MeetingUpdates`.
    in MeetingUpdates (created by WithStyles)
    in WithStyles (created by MeetingUpdatesContainer)
    in MeetingUpdatesContainer (created by Connect(MeetingUpdatesContainer))
    in Connect(MeetingUpdatesContainer) (created by MeetingContainer)
    in div (created by MeetingMain)
    in MeetingMain (created by WithStyles)
    in WithStyles (created by MeetingContainer)
    in div (created by MeetingLayout)
    in MeetingLayout (created by WithStyles)
    in WithStyles (created by MeetingContainer)
    in MeetingContainer (created by Hotkey)
    in Hotkey (created by withRouter(Hotkey))
    in withRouter(Hotkey) (created by Connect(withRouter(Hotkey)))
    in Connect(withRouter(Hotkey)) (created by SocketWithPresence)
    in SocketWithPresence (created by Connect(SocketWithPresence))
    in Connect(SocketWithPresence) (created by SocketClustered)
    in SocketClustered (created by RequiredAuthAndRole)
    in RequiredAuthAndRole (created by withRouter(RequiredAuthAndRole))
    in withRouter(RequiredAuthAndRole) (created by Connect(withRouter(RequiredAuthAndRole)))
    in Connect(withRouter(RequiredAuthAndRole)) (created by RouterContext)
    in div (created by Action)
    in Action (created by WithStyles)
    in WithStyles (created by ActionContainer)
    in ActionContainer (created by RouterContext)
    in RouterContext (created by Router)
    in Router (created by Root)
    in div (created by Root)
    in Provider (created by Root)
    in Root
    in AppContainer
```

I think the `propTypes` need updating. Probably an old bug we hadn't noticed before.